spec/support/shared_examples/dead_jobs.rb

```diff
@@ -59,25 +59,43 @@
 
       expect(csv[0]["register_id"]).to eql trainee.id.to_s
       expect(csv[0]["job_id"]).to eql job_id.to_s
-      expect(csv[0]["error_message"]).to eql "{'title'=>'Teacher has no incomplete ITT record', 'status'=>400, 'errorCode'=>10005}"
+      error_message = csv[0]["error_message"]
+      expect(error_message).to include("title")
+      expect(error_message).to include("Teacher has no incomplete ITT record")
+      expect(error_message).to include("status")
+      expect(error_message).to include("400")
+      expect(error_message).to include("errorCode")
+      expect(error_message).to include("10005")
```

Comment:
Shouldn't those assertions test for key / value pairs?

Reply:
it's the plain text from the csv iirc. The formatting/spacing/quoting seems to have changed a bit between versions ruby versions; this seems a little more robust in case it shifts again